### DISCUSSION TITLE
Has anyone managed to chart COP vs temperature for arotherm just using Vaillant entities?

### DISCUSSION THREAD

**Discussion by willenic** [signalkraft/mypyllant-component#161] [Q&A]
The delay between the actual running of the heatpump and the availability of the yield data is what makes this tricky. Ideally I'd like to build a scatter plot of temp vs COP to draw an efficiency curve to use to schedule my HW runs.

**Comment by Masala-ms** [comment]
Hi @willenic, have you figured out how to do it? I would also be interested in a solution to this.

**Reply by willenic** [reply]
I went down the route of looking at start and end temperature of 0.5 and 1 hour HW runs. Then looking at the variation in power used by ambient temperature. I've sorted the data capture, just need to extend the time range and do some analysis.
